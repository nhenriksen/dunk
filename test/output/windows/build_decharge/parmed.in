parm init.decharge.prmtop
loadcoordinates init.decharge.rst7
timerge :1 :2 :1 :2
parmout decharge.prmtop
writecoordinates decharge.rst7
go
                