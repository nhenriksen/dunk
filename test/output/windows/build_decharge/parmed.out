
  ____    .---.  ,-----.  _____            
.'    `,.':   :'' :    :.'     `.          
| :   :    ' '     '  '   :   :  \         
|  ' '                     ' '    \        
|       v                          \       
|       |       v                   \      
|       |       |       v            \     
|   P   |   A   |   R   |   M   |     \    
|       |       |       |       |      \   
|       |       |       |       |       \  
|       |       |       |       |        \ 
| ===== |       |       |       |         )
 \ --- /| ===== |       |       |        / 
  `---'  \ --- /| ===== | ===== |       /  
          `---'  \ ---- A ----- /      /   
                  `----'.`-----'      /    
                       /             /     
                       '-._    ED   /      
                           '.      /       
                             `----' 

ParmEd: a Parameter file Editor


['parmed.in']
Reading actions from parmed.in

Adding prmtop init.decharge.prmtop to parm list. init.decharge.prmtop is the active parm.
Adding coordinates to init.decharge.prmtop from init.decharge.rst7
Merging molecules [:1] [:2] with sc mask [:1] [:2]
Merging molecules :1 and :2 into the same molecule.
Use softcore mask:
timask1='@1,2,3,4,5,6,7,8,9,10,11,12,13',
timask2='@14,15,16,17,18,19,20,21,22,23,24,25,26',
scmask1='@1,2,3,4,5,6,7,8,9,10,11,12,13',
scmask2='@14,15,16,17,18,19,20,21,22,23,24,25,26',
Writing coordinates to decharge.rst7 as type RESTART
Outputting Amber topology file decharge.prmtop
Done!
